### PR TITLE
Redirect to admin login instead of raising PermissionDenied

### PR DESCRIPTION
**Pull request by wastrachan**
The current behavior of raising a generic `PermissionDenied` exception isn't particularly helpful in a the event of a user's being logged out. I propose redirecting the user to the admin login view if they are logged out or don't have the required permissions. This mirrors the behavior in the rest of the Django Admin. 

In addition, Django designates `is_staff` as the designator for admin site access (1). It and `is_superuser` can be exclusive permissions of one another.

-------------------

1. https://docs.djangoproject.com/en/1.11/ref/contrib/auth/#django.contrib.auth.models.User.is_staff

### CODE REVIEW

File: django_rq_dashboard/views.py [hunk 1]
```diff
@@ -78,8 +78,14 @@ def serialize_scheduled_queues(queue):
 
 class SuperUserMixin(object):
     def dispatch(self, request, *args, **kwargs):
-        if not request.user.is_superuser:
-            raise PermissionDenied
+        if not request.user.is_authenticated():
+            return redirect('{}?{}'.format(reverse('admin:login'),
+                                           urllib.urlencode({
```
Comment (brutasse) [inline django_rq_dashboard/views.py]: This fails on Python 3. Could you update the urllib import to use `django.utils.six.moves.urllib`?

**Review by brutasse — COMMENTED** [review]
Thanks @wastrachan! One nitpick, as this change is Python 2 only :)

File: django_rq_dashboard/views.py (hunk 1 above)
Comment (wastrachan) [inline django_rq_dashboard/views.py]: Duh-- of course. Old habits die hard, sometimes ;)

Comment (brutasse) [inline django_rq_dashboard/views.py]: Thanks :) But then the path becomes `urllib.parse.urlencode` -- you can do `from django.utils.six.moves.urllib.parse import urlencode`

**Comment by brutasse** [comment]
Thanks again :) 0.3.3 is up on PyPI (I messed things up and made a bogus 0.3.2 release so I had to bump the minor version number once more).

**Pull request merged**